packages/mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -19,10 +19,18 @@
         android:launchMode="singleTask"
         android:windowSoftInputMode="adjustResize"
         android:exported="true">
-        <intent-filter>
+        <intent-filter android:autoVerify="true">
```

Comment:
Don't we need `android:autoVerify="true"` to be within the next `intent-filter`?

Reply:
![intent](https://user-images.githubusercontent.com/69375560/189832221-4ca94084-5e37-452e-807a-a59ee4a1a37d.jpg)
This is an example from documentation